profiler/README.MD

```diff
@@ -1,5 +1,12 @@
-﻿# dd-trace-dotnet/profiler
+﻿# dd-trace-dotnet
 
-## Note
+## License
 
-This directory is currently empty. It is a placeholder for the Continuous Profiler source code, which will be available here in the future. In the meantime, the Continuous Profiler is available as a private preview. To inquire about the preview, reach our [support](https://docs.datadoghq.com/help).
\ No newline at end of file
+See [license information](./LICENSE).
```

Comment:
I'm not sure you need to list the license/copyright info since they're available in the root-level README.md. I would also consider moving all of the content from your `../docs/Development.md` into this file, which is what we did. You can see that in `dd-trace-dotnet/docs/README.md` we have a link for the continuous profiler that links to this file

Reply:
✔️ done